src/utils/md.spec.ts

```diff
@@ -77,4 +82,10 @@ describe('utils/md', () => {
   it('includes content', () => {
     assert(cliCompletionClinkUpdateHelpPlain.includes('--verbose'));
   });
+
+  it('converts content tabs with code blocks', () => {
+    assert(plannerPlanAddHelpPlain.includes(`  JSON${EOL}${EOL}  {`));
```

Comment:
Typically, we want one assert per test, so that when it fails, we know which test failed. Right now, if either of the 3 asserts fails, the test will show up as failed, but we won't know why it failed exactly until we debug the test, which isn't ideal. If we want to include multiple asserts, then we should add a custom message for each one so that we can distinguish the different test cases without debugging.